.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,6 +27,18 @@ Tell us about it, and what you did to overcome them!
 Make sure you test your work before opening a PR.
 Include the precise steps to reproduce in order to peer review your work.
 Also include screenshots if you can so that reviewers can compare with a baseline.
+
+ Install Dependencies:- `pnpm install`
+ Build Project:- `pnpm build`
+ Serving To Web:- `pnpm serve`
+ Run Tests:- `npm run test` or `pnpm run test`
+ Check Formatting:- `npm run format` or `pnpm run format`
+ Eslint Check:- `npm run lint:check` or `pnpm run lint:check`
+
+My code follows the project’s style guidelines.
+I have tested my changes thoroughly.
+I have updated relevant documentation.
+
```

Comment:
Just add a small sentence like "Here is a small list of things you can do to check everything is working:" before the actual list

and remove "My code follows the project’s style guidelines. I have tested my changes thoroughly."

and rephrase "I have updated relevant documentation" to "Did you document your code?". I often find asking questions to be more effective.